CONTRIBUTING.md

```diff
@@ -1,118 +1,182 @@
 # Contributing
 
-poliastro is a community project, hence all contributions are more than
-welcome!
+poliastro is a community project, all contributions are more than welcome!
 
-## Bug reporting
+## What you can do
 
-Not only things break all the time, but also different people have
-different use cases for the project. If you find anything that doesn\'t
-work as expected or have suggestions, please refer to the [issue
-tracker](https://github.com/poliastro/poliastro/issues) on GitHub.
+### Report bugs
 
-## Documentation
+Not only things can break, but also different people have
+different use cases for the project. If you find anything that doesn't
+work as expected or have suggestions, open a new issue on our
```

Comment:
What do you think of this?

```suggestion
work as expected or have suggestions, please open a new issue on our
```

Reply:
Good! I removed excessive use of "please", but in this case it makes sense